proxy.conf.json

```diff
@@ -1,45 +1,44 @@
 {
```

Comment:
Changes to this file are not needed. Please remove the same.

Reply:
done